tests/test-runner/bin/zts-report.py

```diff
@@ -265,6 +265,7 @@
     'rsend/rsend_024_pos': ['FAIL', '5665'],
     'rsend/send-c_volume': ['FAIL', '6087'],
     'snapshot/clone_001_pos': ['FAIL', known_reason],
+    'snapshot/rollback_003_pos': ['FAIL', '6143'],
```

Comment:
You will want to add `cli_root/zfs_unshare/zfs_unshare_007_pos` after `cli_root/zfs_unshare/zfs_unshare_006_pos` in the `known` section above since the new test is unconditionally skipped on Linux.

Reply:
Right